wordpress-mobile/WordPress-iOS
# Post published while offline will now be published once back online.

Title has it.

Fixes #11420 

## Testing:

1. Create a post while offline.
2. Tap "Publish".

Expected: Once you are back online make sure the post is published.

## Update release notes:
